docs/source/api.rst

```diff
@@ -348,7 +370,7 @@ Closing a driver will immediately shut down all connections in the pool.
         :returns: the result of the ``result_transformer``
         :rtype: T
 
-        **This is experimental.** (See :ref:`filter-warnings-ref`)
+        **This is experimental** (see :ref:`filter-warnings-ref`).
```

Comment:
Should this be preview?

Reply:
Nope. Since it was marked experimental before and the Python driver emits an `ExperimentalWarning` for those APIs. Changing this to `PreviewWarning` would be a breaking change. So the new wording will only be applied to new preview APIs.

Reply:
Understood.